docs/conf.py

```diff
@@ -26,11 +32,13 @@
 # -- Project information -----------------------------------------------------
 
 project = "xocto"
-copyright = "2023, Kraken Tech"
+copyright = f"{datetime.now().year}, Kraken Tech"
```

Comment:
🐼  I've always wondered.. should we use the current year or the year of first publication? Most things I'm reading at the moment suggest the original year is better, but overall it doesn't matter 🙃